content/en/apps/features/contacts/_index.md

```diff
@@ -10,19 +10,16 @@ relatedContent: >
   apps/reference/contact-page
   apps/guides/updates/moving-contacts
   apps/tutorials/contact-and-users-1
-  apps/tutorials/contact-and-users-2
   apps/examples/contact-tracing
 ---
 <!-- ## Contacts: Person and Family Profiles -->
 <!-- TODO Refine screenshots, and add desktop view. -->
 
 “People” is the generic name used for individuals in apps built with the Core Framework. They can be patients, family members, nurses or health workers. Anyone with a profile in your app is a person.
 
-“Places” is the generic name that represents a level in the hierarchy. “People” belong to “places” and “places” belong to other higher level “places” in the hierarchy.
+“Places” is the generic name that represents a level in the [hierarchy]({{< ref "apps/concepts/hierarchy" >}}). “People” belong to “places” and “places” belong to other higher level “places” in the hierarchy. Depending on the context, a “place” might be a health facility and the “people” who get created at that level might be nurses. Most often for CHWs, these “places” are families. 
```

Comment:
> Most often for CHWs, these “places” are families.

For CHWs there are still multiple level, multiple type of "places". For deployments with CHWs using mobile apps the lowest level of the hierarchy is generally families or households. Here is an attempt to clarify: 

```suggestion
“Places” is the generic name that represents a level in the [hierarchy]({{< ref "apps/concepts/hierarchy" >}}). “People” belong to “places”, and “places” belong to other higher level “places” in the hierarchy. A “place” could be a geographic area, like a district with the "people" associated to it being health officers. A "place" could also be a structure in the health system, such as a health facility, and the "people" associated to it being nurses. In deployments with CHWs, the lowest "place" in the hierarchy often represents individual households or families, and the individual members of that household are the "people" associated to it. 
```